### DISCUSSION TITLE
Language change and customization

### DISCUSSION THREAD

**Discussion by jongoikolea** [NationalSecurityAgency/skills-service#2456] [Q&A]
Hi team! First of all, thank you for the fantastic tool.
I'm considering trying out the platform with my school kids and would like to know if it's possible to translate it and change the logos. How can I achieve this? I'll be using only the dashboard. (Currently, I'm using the Docker installation, if that's relevant.)
Thanks in advance.

**Comment by sudo-may** [comment]
Hi @jongoikolea, thank you so much for the kind words! Unfortunately, as of now, SkillTree Platform does not support language internationalization and localization. As you probably already figured out you can customize Subjects' and Badges' icons but SkillTree logo cannot be changed. Thank yo so much for using SkillTree!
